packages/swirl-components/src/components/swirl-form-control/swirl-form-control.css

```diff
@@ -126,6 +126,21 @@
   @media (--from-desktop-without-touch) {
     --swirl-text-input-placeholder-size: var(--s-font-size-sm);
   }
+
+  & .form-control__label-text {
+    @media (--from-desktop-without-touch) {
+      display: flex;
+      align-items: center;
+    }
+
+    & .form-control__tooltip {
+      display: none;
```

Comment:
**issue:** I cannot see the tooltip, even when i set it and change the label to 'outside'. it is still set to display: none